src/layouts/Search/styled.js

```diff
@@ -3,15 +3,14 @@ import styled from "styled-components";
 export const Wrapper = styled.div`
   max-width: 1368px;
   margin: 0 auto;
-  margin-top: 56px;
+  margin: 56px 16px 0;
```

Comment:
Sorry, this is my faul. In this line should be margin: 56px auto; and padding: 0 16px;.

Reply:
padding added and margin changed